doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -114,7 +114,11 @@ Working with nRF54H Series
 Developing with nRF54L Series
 =============================
 
-* Added the :ref:`ug_nrf54l_cryptography` page, providing more information about the cryptographic peripherals of the nRF54L Series devices, the programming model for referencing keys, and the configuration.
+* Added:
+
+  * :ref:`nRF54l_snippets` to emulate these targets on an nRF54L15 DK.
```

Comment:
@wbober  Do you think we need a statement to be added here that L15 emulated devices are only development purposes.